app/src/test/java/nu/yona/app/utils/MobileNumberTests.java

```diff
@@ -0,0 +1,85 @@
+package nu.yona.app.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MobileNumberTests {
+
+    @Test
+    public void formatPhoneNumber_withSymbols(){
```

Comment:
Follow the pattern ``<method under test>_<input>_<expected outcome>``. E.g. ``format_withSymbols_symbolsStripped``.